### DISCUSSION TITLE
Layout组件api、demo优化讨论

### DISCUSSION THREAD

**Discussion by xingyan95** [DevCloudFE/vue-devui#112] [General]
1. 中间`content`部分的高度是不是太小了？
![image](https://user-images.githubusercontent.com/11143986/147619650-6f4b7216-5d81-413b-9de5-c0c5714040db.png)
2.  `logo`没有展示出来。
![image](https://user-images.githubusercontent.com/11143986/147619655-797a5931-4e83-40fd-b33d-552aecd2ccd5.png)

**Comment by vaebe** [comment]
#1548
